hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/transport/server/ratis/CSMMetrics.java

```diff
@@ -86,7 +107,7 @@ public long getNumWriteStateMachineOps() {
 
   @VisibleForTesting
   public long getNumReadStateMachineOps() {
-    return numReadStateMachineOps.value();
+    return queryStateMachineOps.value();
```

Comment:
Should be numReadStateMachineOps? We need another fn for queryStateMachineOps.

Reply:
Addressed